apidef/api_definitions.go

```diff
@@ -279,8 +279,12 @@ type APIDefinition struct {
 		UseCookie      bool   `mapstructure:"use_cookie" bson:"use_cookie" json:"use_cookie"`
 		CookieName     string `mapstructure:"cookie_name" bson:"cookie_name" json:"cookie_name"`
 		AuthHeaderName string `mapstructure:"auth_header_name" bson:"auth_header_name" json:"auth_header_name"`
+		UseCertificate bool   `mapstructure:"use_certificate" bson:"use_certificate" json:"use_certificate"`
 	} `bson:"auth" json:"auth"`
 	UseBasicAuth            bool                 `bson:"use_basic_auth" json:"use_basic_auth"`
+	UseMutualTLSAuth        bool                 `bson:"use_mutual_tls_auth" json:"use_mutual_tls_auth"`
```

Comment:
just a thought - perhaps we should simply use `len(spec.ClientCertificates) > 0`

Reply:
I wanted to make it consistent with rest of authentification methods. While from technical point of view we can replace it like this, from UX perspective, keeping separate attribute makes more sense, since it follows existing conventions on how we configure authentification.

Reply:
Sounds good - was just a thought.